frame/nfts/src/lib.rs

```diff
@@ -320,16 +361,26 @@ pub mod pallet {
 			owner: T::AccountId,
 			delegate: T::AccountId,
 		},
+		// All approvals of an item got cancelled.
```

Comment:
```suggestion
		/// All approvals of an item got cancelled.
```
and this is the reason why `fmt` formats events differently :)